CDP4Rules/CDP4Rules.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>net45;net451;net452;net46;net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
     <Company>RHEA System S.A.</Company>
     <Title>CDP4Rules Community Edition</Title>
-    <VersionPrefix>7.4.0</VersionPrefix>
+    <VersionPrefix>7.5.0</VersionPrefix>
```

Comment:
i have the impression that nothing has changed in CDP4Rules, please revert

Reply:
Got it. My thought was to increase the version of each of them(projects) in case of a new SDK release.

Reply:
@samatrhea so now with the update of MessageBus in Common I think the version bumps to all libs are necessary.